lte/gateway/c/session_manager/PipelinedClient.cpp

```diff
@@ -29,11 +29,10 @@ magma::SessionSet create_session_set_req(
   std::string subscriber_id                    = info.subscriber_id;
   uint32_t sess_ver_no                         = info.ver_no;
   magma::SessionState::SessionInfo::NodeId tmp = info.nodeId;
-  std::string node_id                          = tmp.node_id;
   req.set_subscriber_id(subscriber_id);
   req.set_session_version(sess_ver_no);
   req.set_local_f_teid(info.local_f_teid);
-  req.mutable_node_id()->set_node_id(node_id);
+  req.mutable_node_id()->set_node_id(tmp.node_id);
```

Comment:
why not directly use `info.nodeID.node_id`?

Reply:
Chaged.